src/main/java/com/synopsys/integration/alert/database/entity/repository/NotificationContentRepository.java

```diff
@@ -45,20 +45,40 @@
-                       + "(definedField.key = '" + ChannelDistributionUIConfig.KEY_CHANNEL_NAME + "' AND LOWER(fieldValue.value) LIKE %:searchTerm% )")
+    //    @Query(value = "SELECT entity FROM NotificationContent entity "
+    //                       + "LEFT JOIN entity.auditNotificationRelations relation ON entity.id = relation.notificationId "
+    //                       + "LEFT JOIN relation.auditEntryEntity auditEntry ON auditEntry.id = relation.auditEntryId "
+    //                       + "LEFT JOIN auditEntry.configGroupEntity configGroup ON auditEntry.commonConfigId = configGroup.jobId "
+    //                       + "LEFT JOIN configGroup.descriptorConfigEntity descriptorConfig ON configGroup.configId = descriptorConfig.id "
+    //                       + "LEFT JOIN descriptorConfig.fieldValueEntities fieldValue ON descriptorConfig.id = fieldValue.configId "
+    //                       + "LEFT JOIN fieldValue.definedFieldEntity definedField ON fieldValue.fieldId = definedField.id "
+    //                       + "WHERE LOWER(entity.provider) LIKE %:searchTerm% OR "
+    //                       + "LOWER(entity.notificationType) LIKE %:searchTerm% OR "
+    //                       + "LOWER(entity.content) LIKE %:searchTerm% OR "
+    //                       + "LOWER(entity.createdAt) LIKE %:searchTerm% OR "
+    //                       + "LOWER(auditEntry.timeLastSent) LIKE %:searchTerm% OR "
+    //                       + "LOWER(auditEntry.status) LIKE %:searchTerm% OR "
+    //                       + "(definedField.key = '" + ChannelDistributionUIConfig.KEY_NAME + "' AND LOWER(fieldValue.value) LIKE %:searchTerm% ) OR "
+    //                       + "(definedField.key = '" + ChannelDistributionUIConfig.KEY_CHANNEL_NAME + "' AND LOWER(fieldValue.value) LIKE %:searchTerm% )")
+    //    Page<NotificationContent> findMatchingNotification(@Param("searchTerm") String searchTerm, final Pageable pageable);
+
+    @Query(value =
+               "SELECT RAW_NOTIFICATION_CONTENT.ID, RAW_NOTIFICATION_CONTENT.CREATED_AT, RAW_NOTIFICATION_CONTENT.PROVIDER, RAW_NOTIFICATION_CONTENT.PROVIDER_CREATION_TIME, RAW_NOTIFICATION_CONTENT.NOTIFICATION_TYPE, RAW_NOTIFICATION_CONTENT.CONTENT "
+                   + "FROM ALERT.RAW_NOTIFICATION_CONTENT "
+                   + "JOIN ALERT.AUDIT_NOTIFICATION_RELATION ON AUDIT_NOTIFICATION_RELATION.NOTIFICATION_ID = RAW_NOTIFICATION_CONTENT.ID "
+                   + "JOIN ALERT.AUDIT_ENTRIES ON AUDIT_ENTRIES.ID = AUDIT_NOTIFICATION_RELATION.AUDIT_ENTRY_ID "
+                   + "JOIN ALERT.CONFIG_GROUPS ON CONFIG_GROUPS.JOB_ID = AUDIT_ENTRIES.COMMON_CONFIG_ID "
+                   + "JOIN ALERT.DESCRIPTOR_CONFIGS ON DESCRIPTOR_CONFIGS.ID = CONFIG_GROUPS.CONFIG_ID "
+                   + "JOIN ALERT.FIELD_VALUES ON FIELD_VALUES.CONFIG_ID = DESCRIPTOR_CONFIGS.ID "
+                   + "JOIN ALERT.DEFINED_FIELDS ON DEFINED_FIELDS.ID = FIELD_VALUES.FIELD_ID "
```

Comment:
Do we want the JOIN to be LEFT JOIN like the old query? or does LEFT JOIN not work in the raw query?